sql/src/test/java/org/apache/druid/sql/avatica/DruidAvaticaHandlerTest.java

```diff
@@ -900,6 +902,39 @@ public void testSqlRequestLog() throws Exception
     Assert.assertEquals(0, testRequestLogger.getSqlQueryLogs().size());
   }
 
+  @Test
+  public void testParameterBinding() throws Exception
+  {
+    PreparedStatement statement = client.prepareStatement("SELECT COUNT(*) AS cnt FROM druid.foo WHERE dim1 = ? OR dim1 = ?");
+    statement.setString(1, "abc");
+    statement.setString(2, "def");
+    final ResultSet resultSet = statement.executeQuery();
+    final List<Map<String, Object>> rows = getRows(resultSet);
+    Assert.assertEquals(
+        ImmutableList.of(
+            ImmutableMap.of("cnt", 2L)
+        ),
+        rows
+    );
+  }
+
+  // this has mocking issues with server discovery and doesn't work, need to fix
```

Comment:
If there's already a tracking issue for the following up work, it can be referenced here.

Optionally, `@Ignore` accepts a parameter for documenting why the test is ignored: http://junit.sourceforge.net/javadoc/org/junit/Ignore.html#value()

Reply:
There is no follow-up issue.. i forgot about this test to be honest 😅. It originally worked but over the life of the PR stuff changed beneath it with mocking on sys table stuff that made it not anymore, and I left this as a note to remind me to fix it or delete it and I've done neither of those things yet. Will do.. something with it, not sure yet.

Reply:
fixed the tests/mocking so there is coverage on using parameters with system tables queries